syntax = "proto3";

message RequestMsg
{
    // 1 秘钥协商   2 秘钥校验    3 秘钥注销
    int32 cmdType = 1;      // 报文类型
    bytes clientId = 2;    // 客户端编号
    bytes serverId = 3;    // 服务器端编号
    bytes sign = 4;
    bytes data = 5;
}

//compile : protoc ./requestmsg.proto --cpp_out=./